src/common/pages/entry.tsx

```diff
@@ -113,17 +133,41 @@ class EntryPage extends BaseComponent<Props, State> {
         }
     }
 
+    componentWillUnmount():void {
+        window.removeEventListener("scroll", this.detect);
+        window.removeEventListener("resize", this.detect);
+    }
+
+    detect = () => {
+
+       const infoCard:HTMLElement | null = document.getElementById("avatar-fixed");
+
+       if(infoCard != null && window.scrollY > 180) {
+            infoCard.classList.add('visible')
+       } else if( infoCard != null && window.scrollY <= 180) {
```

Comment:
also make sure we hide info box when reaching comments section. For example, when we reach comments section on this post, http://localhost:3000/hive-196233/@acidyo/another-post-about-beneficiary-usage we shouldn't show info box in comments...

Reply:
Same for viewing comment itself with direct link, should hide when finished/viewing replies of the post... http://localhost:3000/hive-196233/@acidyo/re-singhcapital-qt32wc

Reply:
Ok I see focusing on it